README.md

```diff
@@ -8,7 +8,7 @@
 
 ## Introduction
 
-SODA Infrastructure Manager is an open source project to provides unified, intelligent and scalable resource management, alert and performance monitoring. It will cover the resource management of all the storage backends & other infrastructures under SODA deployment. It will also provide the alert management and metric data(performance/health) for monitoring and further analysis. It will provide a scalable framework where more and more backends and also client exporters can be added. This will enable to add more storage and infrastructure backends, also support different management clients for monitoring and health prediction.
+delfin (Dolphin in spanish!) , the SODA Infrastructure Manager project is an an open source project to provides unified, intelligent and scalable resource management, alert and performance monitoring. It will cover the resource management of all the storage backends & other infrastructures under SODA deployment. It will also provide the alert management and metric data(performance/health) for monitoring and further analysis. It will provide a scalable framework where more and more backends and also client exporters can be added. This will enable to add more storage and infrastructure backends, also support different management clients for monitoring and health prediction.
```

Comment:
```suggestion
delfin (Dolphin in spanish!) , the SODA Infrastructure Manager project is an an open source project to provide unified, intelligent and scalable resource management, alert and performance monitoring. It will cover the resource management of all the storage backends & other infrastructures under SODA deployment. It will also provide the alert management and metric data(performance/health) for monitoring and further analysis. It will provide a scalable framework where more and more backends as well as client exporters can be added. This will enable to add more storage and infrastructure backends and also support different management clients for monitoring and health prediction.
```